umm-spec-lib/src/cmr/umm_spec/migration/related_url_migration.clj

```diff
@@ -369,3 +369,28 @@
   (if (nil? (:RelatedUrls collection))
     collection
     (update-in-each collection [:RelatedUrls] replace-existing-related-url-keywords "1.10")))
+
+(defn replace-existing-related-url-keywords-for-umm-12
+  "Pass in a related url. If the keywords match what is in the umm-1-11-umm-url-types map then just
+   use the passed in keys because they haven't changed. If they don't match then use the UMM-12 to
+   UMM-11 mapping table."
+  [related-url]
+  (let [{:keys [URLContentType Type Subtype]} related-url
+        keywords (remove-nil-keys {:URLContentType URLContentType
```

Comment:
`url-types` seems to be a better name than `keywords`.

Reply:
fixed